src/PublishCodeCovCoverageTask/utils/webUtils.ts

```
@@ -21,197 +19,109 @@ export async function downloadFile(
-                    if (!redirectUrl) {
-                        return reject(new Error(`Redirect received but no location header for '${fileUrl}'`));
-                    }
-
-                    // Check if we've exceeded the maximum number of redirects
-                    if (redirectCount >= maxRedirects) {
-                        return reject(new Error(`Maximum redirect count (${maxRedirects}) reached for '${fileUrl}'`));
-                    }
 
-                    console.log(`Following redirect (${redirectCount + 1}/${maxRedirects}): ${redirectUrl}`);
+                // Get total size from headers
+                const totalBytes = parseInt(response.headers['content-length'] || '0', 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)